tests/controllers/TalksControllerTest.php

```diff
@@ -775,6 +861,50 @@ public function testRejectClaimAsHostSucceeds()
 
     }
 
+    public function testDifferentTalkMedia()
+    {
+        $request = new Request(
+            [],
+            [
+                'REQUEST_URI' => 'http://api.dev.joind.in/v2.1/talks/3?verbose=yes',
+                'REQUEST_METHOD' => 'GET'
+            ]
+        );
+        $request->user_id = 1;
+        $request->parameters = [
+            'username'      => 'psherman',
+            'display_name'  => 'P Sherman',
+            'verbose'       => 'yes',
+        ];
+        $db = $this->getMockBuilder(mockPDO::class)->getMock();
+        $stmt = $this->getMockBuilder(PDOStatement::class)
```

Comment:
It would be better to inject a mocked TalkMediaMapper into the controller. That way you can test the controller independent from the mapper. That would then also require tests for the TalkMediaMapper.